src/Files.App/Utils/Cloud/CloudDrivesDetector.cs

```diff
@@ -108,6 +108,7 @@ private static Task<IEnumerable<ICloudProvider>> DetectGenericCloudDrive()
 						"Creative Cloud Files" => CloudProviders.AdobeCreativeCloud,
 						"ownCloud" => CloudProviders.ownCloud,
 						"ProtonDrive" => CloudProviders.ProtonDrive,
+						"kDrive" => CloudProviders.kDrive,
```

Comment:
Don't you have to add this?

```c#
if (driveType.StartsWith("kDrive"))
	driveType = "kDrive";
```

Reply:
I don't think this is necessary because the ApplicationName in the registry is exactly kDrive
![image](https://github.com/user-attachments/assets/3720c77e-ea06-4893-a5c8-e46f96e48348)

For the others (e.g. iCloudDrive and OneDrive) there is sometimes an ID or name behind it
![image](https://github.com/user-attachments/assets/483a5f2f-57d8-4069-87fa-da388bcf32a6)
![image](https://github.com/user-attachments/assets/2a62cfb6-c439-4cc2-9e63-399372b87140)

Reply:
Thanks for letting me know!